presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainFilter.java

```diff
@@ -1098,7 +1108,7 @@ private BytesValues(byte[][] values, boolean nullAllowed)
             super(true, nullAllowed);
 
             requireNonNull(values, "values is null");
-            checkArgument(values.length > 1, "values must contain at least 2 entries");
+            checkArgument(values.length > 0, "values must contain at least 1 entries");
```

Comment:
message needs updating: `checkArgument(values.length > 0, "values must not be empty");`